### PR TITLE
Add final code check link & confetti to tutorial completion page

### PR DESCRIPTION
**Pull request by delucis**
#### What kind of changes does this PR include?

- New or updated content
- Changes to the docs site code

#### Description

Updates to the tutorial completion page:

- Add a link to the final version of the tutorial code on GitHub and StackBlitz

- Add confetti on page load to celebrate 🎊

### CODE REVIEW

File: src/components/tutorial/CompletionConfetti.astro [hunk 1]
```diff
@@ -0,0 +1,24 @@
+<script>
+	if (
+		window.location.pathname.endsWith('/tutorial/6-islands/3/') &&
```
Comment (tony-sull) [inline src/components/tutorial/CompletionConfetti.astro:3]: is there an easy way to compute the last tutorial step in case more steps are added later?

Comment (tony-sull) [inline src/components/tutorial/CompletionConfetti.astro:3]: Can the pathname check be left out since it's added as a component directly to the page?

Comment (delucis) [reply src/components/tutorial/CompletionConfetti.astro:3]: Ah, I should probably have added a comment explaining this slightly eccentric code — will do that now.

We glob these pages so this script will end up everywhere. Added the check to target a specific page and avoid downloading the confetti lib unnecessarily (hence the dynamic import).

**Comment by sarah11918** [comment]
Obviously I love this! Though, not to be the no fun, stickler here.... but I think we discussed the confetti flying when you completed the final checklist?  😅 

More seriously, for continuity, it maybe seems a little off to be reading, "There's only one edit left to make.." and before you even get that far, confetti explodes in your face!  😄   How difficult would it be to tie it to the last checkmark, or at least wait until you've scrolled to the bottom of the page?

**Comment by delucis** [comment]
Thanks for the idea @sarah11918! The confetti now only shows when you check the final checkbox off. Added bonus: you can uncheck and recheck as much as you want for extra confetti 🎉

**Comment by sarah11918** [comment]
Preview looks great! I think this is an amazing add to this page!

**Pull request merged**